### PR TITLE
Repository Migration

### PR DESCRIPTION
**Pull request by genebean**
This PR updates things to reflect their new home in grafana/puppet-promtail.

To do in this PR:

- [x] Add a LICENSE file
- [x] Update metadata.json
- [x] Update README.md
- [x] Migrate to using Drone for CI

### CODE REVIEW

**Comment by genebean** [comment]
Please note: the labels have been updated on this repository using https://github.com/underscorgan/community_management and the following:

```
 ☕️  gene:~/repos/community_management (master ✔)
╚ᐅ bundle exec ruby labels.rb -n grafana -r '^puppet-promtail' -f -d
Checking for the following labels: ["needs-squash", "needs-rebase", "needs-tests", "needs-docs", "bugfix", "feature", "tests-fail", "backwards-incompatible", "maintenance"]
Delete: grafana/puppet-promtail, ["bug", "documentation", "duplicate", "enhancement", "good first issue", "help wanted", "invalid", "question", "wontfix"]
Create: grafana/puppet-promtail, [{:name=>"needs-squash", :color=>"bfe5bf"}, {:name=>"needs-rebase", :color=>"3880ff"}, {:name=>"needs-tests", :color=>"ff8091"}, {:name=>"needs-docs", :color=>"149380"}, {:name=>"bugfix", :color=>"00d87b"}, {:name=>"feature", :color=>"222222"}, {:name=>"tests-fail", :color=>"e11d21"}, {:name=>"backwards-incompatible", :color=>"d63700"}, {:name=>"maintenance", :color=>"ffd86e"}]
Fix: grafana/puppet-promtail, []
```

The "why" for this can be seen at https://github.com/ploperations/repo_standards#label-setup

**Comment by genebean** [comment]
We should avoid merging until CI has been enabled and run for this repo. I have added a `.drone.yml` file that _should_ do what's needed.

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+kind: pipeline
+name: ruby-2-5-3
+
+steps:
+  - name: pupept6
```
Comment (pracucci) [inline .drone.yml]: I guess you mean `puppet6` here 😉

Comment (genebean) [reply .drone.yml]: oops... fixed :)

**Pull request merged**